src/main/resources/templates/application/prior-authority-details.html

```diff
@@ -0,0 +1,131 @@
+                Prior authority type
+              </dt>
+              <dd class="govuk-summary-list__value"  th:text="${priorAuthorityFlow.priorAuthorityTypeFormDataDetails.priorAuthorityTypeDisplayValue}">
+              </dd>
+            </div>
+          </dl>
+
+          <div th:replace="~{partials/forms :: textInput('summary', 'Summary', 'govuk-input--width-20')}"></div>
+          <div th:replace="~{partials/forms :: largeTextInput('justification', 'Justification')}"></div>
+
+          <th:block th:if="*{valueRequired}">
+            <div th:replace="~{partials/forms :: currencyInput('amountRequested', 'Amount requested', '10')}"></div>
+          </th:block>
+
+          <th:block th:each="option : ${priorAuthorityDynamicForm.priorAuthorities}">
+            <div class="govuk-form-group" th:classappend="${#fields.hasErrors('dynamicOptions[__${option.code}__].fieldValue')} ? 'govuk-form-group--error' : ''">
+
+                <th:block th:if="${option.mandatoryFlag}">
+                  <label class="govuk-label" th:for="'dynamicOptions[' + ${option.code} + '].fieldValue'" th:text="${option.description}">
+                  </label>
+                </th:block>
+                <th:block th:unless="${option.mandatoryFlag}">
+                  <label class="govuk-label" th:for="'dynamicOptions[' + ${option.code} + '].fieldValue'" th:text="${option.description + ' (optional)'}">
+                  </label>
+                </th:block>
+                <span th:if="${#fields.hasErrors('dynamicOptions[__${option.code}__].fieldValue')}"
+                      th:errors="*{dynamicOptions[__${option.code}__].fieldValue}" class="govuk-error-message">
+                </span>
+              <th:block  th:if="${option.dataType == 'LOV'}">
+                <input type="hidden" th:id="'dynamicOptions[' + ${option.code} + '].fieldValueDisplayValue'"
+                       th:name="'dynamicOptions[' + ${option.code} + '].fieldValueDisplayValue'"
+                       th:value="*{dynamicOptions[__${option.code}__].fieldValueDisplayValue}">
+                <select class="govuk-select"
+                        data-module="govuk-select"
+                        th:id="'dynamicOptions[' + ${option.code} + '].fieldValue'"
+                        th:field="*{dynamicOptions[__${option.code}__].fieldValue}"
+                        th:errorclass="govuk-select--error"
+                        th:attr="onchange=|document.getElementById('dynamicOptions[${option.code}].fieldValueDisplayValue').value = this.options[this.selectedIndex].text;|">>
+                  <option value="">Please Select</option>
+                  <option th:each="item : ${__${option.code}__}"
+                          th:value="${item['code']}"
+                          th:text="${item['description']}">
+                  </option>
+                </select>
+              </th:block>
+              <th:block th:if="${option.dataType == 'FTS'}">
```

Comment:
Can we extract this into a YAIM file?

;-)

Reply:
Anything you want me to do here? Not sure what else we would do?

Reply:
Ouch. Was a ROOF joke. I will resolve this conversation immediately.